cocos/native-binding/index.ts

```diff
@@ -361,6 +353,23 @@ export declare namespace native {
         }
     }
 
+    /**
+      * @en DownloadTask @zh 下载任务对象
+      * @param requestURL @en Request download resource URL  @zh 请求下载资源的URL
+      * @param storagePath @en Storage path for downloaded file @zh 下载文件存储路径
+      * @param identifier  @en identifier @zh 标识符
+      */
+    export type DownloadTask = { requestURL: string, storagePath: string, identifier: string };
+
+    /**
+     * @en DownloaderTask @zh 下载任务对象
+     * @param requestURL @en Request download resource URL  @zh 请求下载资源的URL
+     * @param storagePath @en Storage path for downloaded file @zh 下载文件存储路径
+     * @param identifier  @en identifier @zh 标识符
+     * @deprecated since v3.7.0, please use `DownloadTask` to instead.
+     */
+    export type DownloaderTask = { requestURL: string, storagePath: string, identifier: string };
```

Comment:
@PPpro please check if it is correct way to deprecated a function.

Reply:
scripts will show strikethrough when use ~~DownloaderTask~~ but run nomrally.